scripts/tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Following `core` CI style, let's have this in the `CI` itself, with the way CI sets environment and so

Reply:
Do you mean like so?
https://github.com/jina-ai/executor-image-normalizer/pull/2/commits/ccc42a707286e1aa14974029f54621abd59c2bf8